### DISCUSSION TITLE
ESM enabled or not in @quick-start/electron?

### DISCUSSION THREAD

**Discussion by hyperknot** [alex8088/electron-vite#630] [Q&A]
When using @quick-start/electron, I see that the imports are ESM style, but the package.json is missing "type": "module".

I cannot do something like:

```
await fs.mkdir()
```

on a top level of an import, as it tells me 

```
 ERROR  Module format "cjs" does not support top-level await. Use the "es" or "system" output formats rather.
```

My question is there anything else I need to do to enable ESM modules, or adding `"type": "module"` to package.json is enough? Is there any reason why this is not the default in @quick-start/electron?

**Comment by AhmedBaset** [comment]
Did you find a solution?

**Reply by alex8088** [reply]
https://electron-vite.org/guide/dev#esm-support-in-electron
